apiserver/params/network.go

```diff
@@ -827,7 +827,7 @@ type NetworkInfoResultsV6 struct {
 type NetworkInfoParams struct {
 	Unit       string   `json:"unit"`
 	RelationId *int     `json:"relation-id,omitempty"`
-	Bindings   []string `json:"bindings"`
```

Comment:
Why change the variable name without changing the string-on-the-wire ? I was at first worried about compatibility, but since you didn't change the bytes, it doesn't change the API, but it does add confusion if someone was trying to understand.
At a *minimum* we need a comment as to why the json serialization doesn't match.

Reply:
I missed this one when I landed. I'll address it soon.